core/src/main/java/tc/oc/pgm/kits/KitParser.java

```diff
@@ -413,7 +413,14 @@ public ItemStack parseItem(Element el, Material type) throws InvalidXMLException
   }
 
   public ItemStack parseItem(Element el, Material type, short damage) throws InvalidXMLException {
-    int amount = XMLUtils.parseNumber(el.getAttribute("amount"), Integer.class, 1);
+    int amount;
+    String amountString = el.getAttributeValue("amount");
+    if (amountString != null && amountString.equals("oo")) {
+      // blocks with -1 items cannot be dropped, stacked or dragged/split
+      amount = -1;
+    } else {
+      amount = XMLUtils.parseNumber(el.getAttribute("amount"), Integer.class, 1);
+    }
```

Comment:
```suggestion
    int amount = XMLUtils.parseNumber(el.getAttribute("amount"), Integer.class, true, 1);
    if (amount = Integer.MAX_VALUE) amount = -1;
```

Reply:
I don't understand this solution, parseNumber will return its own XMLDefinition error when it tries to parse `oo`, which is why I've made the separate String variable to parse it.

Reply:
`parseNumber` has multiple overloads (same name, different input parameters), if you use the one with an `infinite` parameter and pass in a `true`, it supports parsing `oo` as infinite, and returns Integer.MAX_VALUE for infinite integer (for doubles/floats there's a specific representation of infinity)